### ISSUE TITLE
Hi, why novel views synesis result not good?

### ISSUE THREAD

**Issue by jinfagang** [philgras/neural-head-avatars#9]
Here is the train-0_0:

https://user-images.githubusercontent.com/21303438/166866110-4465ec84-6b0f-4d4d-bee5-31f9c90252fb.mp4

Here is the train - 30_0:

https://user-images.githubusercontent.com/21303438/166866126-f0c996ad-54b3-4bd7-880d-e2cdcbc71463.mp4

as you can see, from novel view, the result is far from satisfying.

**Comment by MorganPhobo5** [comment]
Hi! Did you change number of epochs? I had same results in case of insufficient number of epochs.

**Comment by jinfagang** [comment]
@MorganPhobo5 I just changed batch size. Didn't change epochs. Does default epochs not enough? If so, how many does it needed?

**Comment by philgras** [comment]
Hi,
The number of epochs was tuned for our sequences with around 750 training frames. When you use more/less frames, you might decrease/increase the number of epochs in order to get roughly the same number of updates. If you decrease the batch_size because of memory limitations, you probably want to adapt the learning rate by a similar factor, as well.

Anyway, regarding your main problem. The reason, why you encounter such bad side view results, is that within the training sequence the subject's head never turns and consequently those information are not captured. The other problem is that the geometry reconstruction fails. This is most likely due to noisy preprocessing outputs (see the explanation in #4 ).

Please check out the example videos, we published with the code as an example for what movements an input video should record.
Hope this helps!
